### PR TITLE
Dckm 432 wallet sdk integration tests

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: integration-tests/accounts.test.ts [hunk 1]
```diff
@@ -11,11 +11,14 @@ import {
...
 
 describe('Accounts', () => {
+  let wallet;
+
   beforeAll(async () => {
     await cleanup();
     await setupEnvironent();
     await createNewWallet();
     await createAccounts();
+    wallet = await getWallet();
```
Comment (mike-parkhill) [inline integration-tests/accounts.test.ts:21]: do we need to close this?

Comment (maycon-mello) [reply integration-tests/accounts.test.ts:22]: yes, should be fixed

File: .github/workflows/integration-tests.yaml [hunk 2]
```diff
@@ -6,40 +6,68 @@ concurrency:
...
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          PR_TITLE: ${{ github.event.pull_request.title }}
 
-      - name: Install dependencies
-        run: yarn install
+      - name: Debug GitHub Context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo "$GITHUB_CONTEXT"
+      # - name: Cache dependencies
```
Comment (mike-parkhill) [inline .github/workflows/integration-tests.yaml:38]: are we keeping these commented out steps?

File: scripts/slack-reporter.js [hunk 3]
```diff
@@ -56,11 +60,20 @@ class SlackReporter {
           text: `<${process.env.GITHUB_ACTION_URL}|View Run in Github>`,
         },
       },
+      {
+        type: 'section',
+        text: {
+          type: 'mrkdwn',
+          text: `<${process.env.PR_LINK}|Pull Request>`,
```
Comment (mike-parkhill) [inline scripts/slack-reporter.js:67]: let's include the PR number in the text here

**Pull request merged**